src/index.js

```diff
@@ -0,0 +1,48 @@
+import './style.css';
+import Task from './task-class.js';
+import Dots from './images/dots.png';
+
+const task1 = new Task(0, 'Attend morning session meeting', false);
+const task2 = new Task(1, 'Meet coding partners', false);
+const task3 = new Task(2, 'Attend morning session meeting', false);
+const tasks = [
+  {
+    index: task1.index,
+    description: task1.desc,
+    completed: task1.completed,
+  },
+  {
+    index: task2.index,
+    description: task2.desc,
+    completed: task2.completed,
+  },
+  {
+    index: task3.index,
+    description: task3.desc,
+    completed: task3.completed,
+  },
+];
```

Comment:
- Please make sure you are implementing all [project requirements:](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md)
- [ ]  you still have hardcoded items in array. No add, edit and remove method implemented.
- [ ] Also make sure all changes to TO DO list are saved in local storage
- [ ] Please make sure your readme file is updated with all required features.

Reply:
I am going to work on it right away. Thanks